### DISCUSSION TITLE
Building Electron App - app.asar

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#10799] [General - Components / Directives / etc] [answered]
When I build my Electron app, it seems to place all my static resources into a asar file? Is there a way I can prevent that combination of files and just have the files losely live in the resources directory. The intent is to make these files available to the user for editing but they can't if it's in this odd compressed file.

I'm assuming there are options or parameters i need to adjust to prevent this from happening. Otherwise what other solutions are recommended for having resource files ship with the application that are meant to remain in native file formats. 

Right now I access these during dev with this `const fp = path.join(remote.app.getAppPath(), "json/config.json");` 

however i want this json file to be in the resources folder at build time as a sample file for the users to edit or copy and modify. I'm using Quasar v2.

**Comment by JokerMartini** [comment]
Any ideas on how to achieve this?

**Comment by ontwikkelfabriek** [comment]
@JokerMartini 

Qv1 or Qv2?

**Reply by JokerMartini** [reply]
I'm using Q2

**Comment by ontwikkelfabriek** [comment]
@JokerMartini 

> I am using Qv2

For Qv2, recourses put in the `public` folder will remain `unprocessed`. I cannot confirm this for Electron builds but that's what the doc says:
https://quasar.dev/quasar-cli/handling-assets#static-assets-public

**Comment by JokerMartini** [comment]
Still have not had any luck. Does anyone know how i can access the path to static resources and not have them be compiled into the app.asar file? I need my resources to be the original file?

Build
`remote.app.getAppPath() >> C:\Users\user0\Documents\HyvElectron\dist\electron\App-win32-x64\resources\app.asar`

Dev
'remote.app.getAppPath() >>  C:\Users\user0\Documents\HyvElectron\.quasar\electron'

**Comment by pdanpdan** [comment, marked as answer]
In `quasar.conf.js`:

```javascript
  electron: {
    bundler: 'packager', // 'packager' or 'builder'

    packager: {
      ...
      asar: {
        unpack: '**/node_modules/{prefix1,prefix2}-something/**',
      }
    }
    ...
  }
```

Then to get the real path:
`standardPathWithAsar.replace('app.asar', 'app.asar.unpacked')`

For builder I don't know.
